wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -227,6 +227,9 @@ <h1>{{ "COMMON.SEARCH.NO_ADDONS" | translate }}</h1>
     <button mat-menu-item (click)="onShowFolder(listItem.addon)">
       {{ "PAGES.MY_ADDONS.ADDON_CONTEXT_MENU.SHOW_FOLDER" | translate }}
     </button>
+    <button mat-menu-item (click)="onVisitProvider(listItem.addon)">
```

Comment:
Can we drop this click handler and make it the same as the details link? If we make it an a and not a button it should look the same. Note the appExternalLink as well.

https://github.com/WowUp/WowUp/blob/develop/wowup-electron/src/app/components/addon-detail/addon-detail.component.html#L33

Reply:
I've converted this button to an a